Calendar/Calendar.js

```diff
@@ -1,3 +1,211 @@
 /**
  * Sandstone styled calendar component and behavior.
+ *
+ * @example
+ * <Calendar />
+ *
+ * @module sandstone/Calendar
+ * @exports Calendar
+ * @private
  */
+
+import kind from '@enact/core/kind';
+import Spottable from '@enact/spotlight/Spottable';
+import {Cell, Row} from '@enact/ui/Layout';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+
+import BodyText from '../BodyText';
+import Button, {ButtonBase} from '../Button';
+import Dropdown from '../Dropdown';
+import Skinnable from '../Skinnable';
+
+import componentCss from './Calendar.module.less';
+
+const SpottableButton = Spottable(ButtonBase);
+
+/**
+ * The calendar base component which sets-up the component's structure.
+ *
+ * This component is most often not used directly but may be composed within another component as it
+ * is within {@link sandstone/Calendar|Calendar}.
+ *
+ * @class CalendarBase
+ * @memberof sandstone/Calendar
+ * @ui
+ * @private
+ */
+
+const CalendarBase = kind({
```

Comment:
I would say to refactor it to a React functional component. I foresee multiple states and useCallback